apps/meteor/client/views/omnichannel/analytics/Overview.tsx

```diff
@@ -62,7 +63,7 @@ const Overview = ({ type, dateRange, departmentId }) => {
 							pb='x8'
 							flexBasis='100%'
 							key={i}
-							title={title ? t(title) : <Skeleton width='x60' />}
+							title={title ? t(title as TranslationKey) : <Skeleton width='x60' />}
```

Comment:
Can we properly type this to avoid the typecast? Maybe we can pass a generic to the useState or typing the conversationsInitialData var.

wdyt @aleksandernsilva ?

Reply:
Also, on L58, I see there's a default value for a map's first element. Is there any special case in which that default will be triggered? 👀